packages/fxa-event-broker/lib/grant.ts

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as jwtool from 'fxa-jwtool';
+
+// SET Event identifiers
+export const SUBSCRIPTION_STATE_EVENT_ID =
+  'https://schemas.accounts.firefox.com/event/subscription-state-change';
+
+type securityEvent = {
+  uid: string;
+  clientId: string;
+  events?: {
+    [key: string]: {
+      [key: string]: any;
+    };
+  };
+};
+
+type subscriptionEvent = {
+  uid: string;
+  clientId: string;
+  capabilities: string[];
+  isActive: boolean;
+};
+
+type JWTConfig = {
+  issuer: string;
+  key: any;
+};
+
+export class JWT {
+  private issuer: string;
+  private tokenKey: jwtool.PrivateJWK;
+
+  constructor(config: JWTConfig) {
+    this.issuer = config.issuer;
+    this.tokenKey = jwtool.JWK.fromObject(config.key, { iss: this.issuer });
+  }
+
+  /**
+   * Generate a Security Event Token for delivery to Relying Parties.
+   *
+   * See: https://tools.ietf.org/html/rfc8417
+   *
+   * @param grant
+   */
+  public generateSET(grant: securityEvent): Promise<string> {
```

Comment:
as above, "grant" doesn't seem like the right name in this context; "claims" would be the generic name for "the things I want to put inside a JWT" but "event" would probably work fine in this more specific context.